services/dy-jupyter-extensions/neuron/docker/boot.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+# BOOTING application ---------------------------------------------
+echo "Booting application ..."
+echo "  User    :`id $(whoami)`"
+echo "  Workdir :`pwd`"
+
+# start the notebook now
+echo "Setting theme ..."
+jt -t grade3 -f ubuntu -fs 12 -T -N -cellw 90%
```

Comment:
shall we move that to the base?

Reply:
I do not know, I like the 90% thing

Reply:
I mean to the jupyter-base, that would probably also be beneficial there

Reply:
I move it to the base. But no everything is bright and with ubuntu fonts... Let the PO's decide...